### DISCUSSION TITLE
Server Error - class 'ImportError' - cannot import name 'CableTermination' from 'dcim.models'

### DISCUSSION THREAD

**Discussion by rommeljantanap** [netbox-community/netbox#10299] [General]
I have this error right after I tried to install the topology viewer (I followed the instructions from https://github.com/mattieserver/netbox-topology-views)

Server Error
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'ImportError'>

cannot import name 'CableTermination' from 'dcim.models' (/opt/netbox/netbox/dcim/models/__init__.py)

Python version: 3.10.4
NetBox version: 3.2.7

**Comment by rommeljantanap** [comment]
I am unable  to make the Plugin (netbox-topology-view) work

**Reply by candlerb** [reply]
Are you using the right [version](https://github.com/mattieserver/netbox-topology-views#versions) of netbox-topology-views for the version of Netbox you're using?

**Reply by rommeljantanap** [reply]
How do you specify in the install which version of the netbox-topology-views to use?

**Reply by candlerb** [reply]
In local_requirements.txt:
```
netbox-topology-views==1.1.0
```
(or 1.2.0 or 2.0.3 or whichever version you want to try). Then re-run `./upgrade.sh`

Having said that: it does say version ">= 1.1.0" which suggests that the author intends that the latest released version *should* run on all older versions of Netbox.  But the error says it's trying to import a model which only exists in Netbox 3.3.

**Reply by rommeljantanap** [reply]
Thank you for the information. It works like a charm.
